samples/fbuffer/enums.hpp

```diff
@@ -0,0 +1,23 @@
+/// \file     enums.hpp
+/// \authors  ggonzalez
+/// \version  1.0
+/// \date     03/06/20
+/// \license  GPLv3
+/// \brief    Copyright (c) 2018 Advens. All rights reserved.
+
+#pragma once
+
+typedef enum valueType_e {
+    STRING = 0,
+    INT,
+    DOUBLE,
+    FLOAT,
+    UNKNOWN
+} valueType;
+
+typedef enum outputType_e {
+    ANOMALY = 0,
+    SOFA,
+    BUFFER
+} outputType;
+
```

Comment:
Could you put those enums in a namespace ?
Is it possible to include them into another class/file ?